### PR TITLE
154052162 duplicate service problem

### PR DESCRIPTION
**Pull request by rckopteri**
I had to do some work with our forms and grid. Now all forms work much better in mobile views.
I fixed that duplicate bug by adding a check that if a save event is on going then another won't be possible to send.

### CODE REVIEW

File: ote/src/cljs/ote/app/controller/transport_service.cljs [hunk 1]
```diff
@@ -299,10 +299,18 @@
...
+      ;; Disable post if concurrent save event is in progress
+      (if (not (:service-save-in-progress app))
+        (do
+          (comm/post! "transport-service" service-data
+                      {:on-success (tuck/send-async! ->SaveTransportServiceResponse)
+                       :on-failure (tuck/send-async! ->FailedTransportServiceResponse)})
+          (-> app
+              (assoc :service-save-in-progress true)
+              (dissoc :before-unload-message))
+          )
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/app/controller/transport_service.cljs]: fix bracket position

File: ote/src/cljs/ote/views/rental.cljs [hunk 2]
```diff
@@ -279,7 +280,7 @@
       :add-label (tr [:buttons :add-new-pick-up-location])})))
 
 (defn rental [e! service]
-  (reagent/with-let [groups [(ts-common/name-group (tr [:rentals-page :header-service-info]))
+  (with-let [groups [(ts-common/name-group (tr [:rentals-page :header-service-info]))
                              (ts-common/contact-info-group)
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/views/rental.cljs]: fix indentation

Comment (rckopteri) [reply ote/src/cljs/ote/views/rental.cljs]: fixed

File: ote/src/cljs/ote/app/controller/transport_service.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/transport_service.cljs]: fixed

**Pull request merged**